ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellFileAccessoryViewDemoController.swift

```diff
@@ -3,117 +3,416 @@
+            case .showErrorButton:
+                return #selector(toggleErrorButton(_:))
+            case .showErrorButtonOnTopCellOnly:
+                return #selector(toggleErrorOnBottomCellButton(_:))
+            case .showOverflowButton:
+                return #selector(toggleOverflow(_:))
+            }
+        }
 
-        for width in Constants.cellWidths {
-            if !useDynamicWidth {
-                let cellTitle = Label(style: .subhead, colorStyle: .regular)
-                cellTitle.text = "\t\(Int(width))px"
-                stackView.addArrangedSubview(cellTitle)
+        var isOn: Bool {
+            switch self {
+            case .dynamicWidth,
+                 .showDate,
+                 .showSharedStatus,
+                 .isDocumentShared,
+                 .showKeepOfflineButton,
+                 .showShareButton,
+                 .showPinButton,
+                 .showOverflowButton:
+                return true
+            case .dynamicPadding,
+                 .chooseDate,
+                 .chooseTime,
+                 .minimumActionsCount,
+                 .topActionsOverlap,
+                 .bottomActionsOverlap,
+                 .disableShareButton,
+                 .disablePinButton,
+                 .showErrorButton,
+                 .showErrorButtonOnTopCellOnly:
+                return false
             }
+        }
+    }
 
-            let cell1 = createCell(title: "Document Title", subtitle: "OneDrive - Microsoft · Microsoft Teams Chat Files", top: true)
-            let cell2 = createCell(title: "This is a very long document title that keeps on going forever to test text truncation",
-                                   subtitle: "This is a very long document subtitle that keeps on going forever to test text truncation", top: false)
+    private lazy var plusMinActionsButton: UIButton = createPlusMinusButton(plus: true, #selector(incrementMinimumActionsCount))
+    private lazy var minusMinActionsButton: UIButton = createPlusMinusButton(plus: false, #selector(decrementMinimumActionsCount))
+    private lazy var plusTopOverlapButton: UIButton = createPlusMinusButton(plus: true, #selector(incrementTopActionsOverlap))
+    private lazy var minusTopOverlapButton: UIButton = createPlusMinusButton(plus: false, #selector(decrementTopActionsOverlap))
+    private lazy var plusBottomOverlapButton: UIButton = createPlusMinusButton(plus: true, #selector(incrementBottomActionsOverlap))
+    private lazy var minusBottomOverlapButton: UIButton = createPlusMinusButton(plus: false, #selector(decrementBottomActionsOverlap))
 
-            let containerView = UIStackView(frame: .zero)
-            containerView.axis = .vertical
-            containerView.translatesAutoresizingMaskIntoConstraints = false
+    private func createPlusMinusButton(plus: Bool, _ selector: Selector) -> UIButton {
+        let button = Button(style: .secondaryOutline)
+        button.setTitle(plus ? "+" : "-",
```

Comment:
we shouldn't use these text right? wouldn't it weirdly increase in larger text mode? just use the image that is already in demo app asset catalog.

Reply:
also, nit, if we follow apple hig for stepper minus is left and plus is right side.